### ISSUE TITLE
Proposal Node geometric progression

### ISSUE THREAD

**Issue by satabol** [nortikin/sverchok#4960]
Sverchok . No node for geometric progression

![image](https://github.com/nortikin/sverchok/assets/14288520/991a4881-9186-4a2c-8cbd-adb08a2aa8c5)

Except general purpose it is may be used for geometric parametrization along with node "Number Range". Proposal parameters:

Base parameters:
- base
- q
- count

additional params

- scale (scale all value in range)
- start (start value before base: Range=>start + scale * b0, start + scale * b1...)

Output:
Range: [b0, b1, b2]
steps: [b1-b0, b2-b1, b3-b2]

Some parameters viewed strange but this is example of service (it may change).

**Comment by satabol** [comment]
Clean geometric progression is n node “Exponentional sequence”:
http://nortikin.github.io/sverchok/docs/nodes/number/exponential.html
but no cumulative summ in it.

**Comment by Durman** [comment]
For accumulative sum there is another node:

![image](https://github.com/nortikin/sverchok/assets/28003269/6c949d9a-9de4-47d3-b27e-aa60c1dda2fd)

**Comment by satabol** [comment]
You are right but if you know this node and numpy. But the target is take maximum capabilities with generator nodes. One node generate cumulative params and the generator create objects by cumulative params. Just a pair of nodes.

![image](https://github.com/nortikin/sverchok/assets/14288520/06b1b1ec-1547-4691-a258-fe983e420db4)
![image](https://github.com/nortikin/sverchok/assets/14288520/20c51137-770f-4cf5-a984-6579595e3448)

**May be extend "Exponention sequence"?**

![image](https://github.com/nortikin/sverchok/assets/14288520/19522d16-d28a-4ef2-b9f3-83b19fcefcda)
